src/metabase/startup.clj

```diff
@@ -0,0 +1,32 @@
+(ns metabase.startup
```

Comment:
learned this morning we want to put new modules in `core` namespaces inside a folder: https://metaboat.slack.com/archives/CKZEMT1MJ/p1745962074021709